src/ots.h

```diff
@@ -187,9 +187,12 @@ bool IsValidVersionTag(uint32_t tag);
 #define OTS_TAG_CFF  OTS_TAG('C','F','F',' ')
 #define OTS_TAG_CMAP OTS_TAG('c','m','a','p')
 #define OTS_TAG_CVT  OTS_TAG('c','v','t',' ')
+#define OTS_TAG_FEAT OTS_TAG('F','e','a','t')
```

Comment:
I think we will need to make the tag macros case sensitive if we ever supported AAT tables (which include a `feat` table all lowercase), but lets worry about this when it happens.